flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/LocalInputChannel.java

```diff
@@ -336,6 +336,16 @@ void releaseAllResources() throws IOException {
         }
     }
 
+    @Override
+    void announceBufferSize(int newBufferSize) {
+        // Not supported.
```

Comment:
I will implement it in the following PR. Right now we don't have the right methods inside of PipelinedSubpartition